src/modules/data/blocks/ticket/sagas.js

```diff
@@ -163,32 +163,36 @@ export function* setTicketInitialState( action ) {
 		put( actions.setTicketHasBeenCreated( clientId, hasBeenCreated ) ),
 	] );
 
-	try {
-		// NOTE: This requires TEC to be installed, if not installed, do not set an end date
-		const eventStart = yield select( tribe.events.data.blocks.datetime.selectors.getStart ); // Ticket purchase window should end when event starts
-		const endMoment = yield call( momentUtil.toMoment, eventStart );
-		const endDate = yield call( momentUtil.toDatabaseDate, endMoment );
-		const endDateInput = yield datePickerFormat
-			? call( momentUtil.toDate, endMoment, datePickerFormat )
-			: call( momentUtil.toDate, endMoment );
-		const endTime = yield call( momentUtil.toDatabaseTime, endMoment );
-		const endTimeInput = yield call( momentUtil.toTime, endMoment );
+	const isEvent = yield call( isTribeEventPostType );
+
+	if ( isEvent && window.tribe.events ) {
```

Comment:
Was this not working before? Or was this running in non-event post types? The `window.tribe.events` and the `try-catch` are redundant.

Reply:
This was running on non-event post types. I thought the catch would be useful to see unexpected errors, this would be an expected bypass

Reply:
done